lib/application.ts

```diff
@@ -1193,7 +1203,7 @@ export class SNApplication {
         if (mergeLocal) {
           await this.syncService.markAllItemsAsNeedingSync(true);
         } else {
-          this.itemManager.removeAllItemsFromMemory();
+          void this.itemManager.removeAllItemsFromMemory();
```

Comment:
Explain?

Reply:
`@typescript-eslint/no-floating-promises` will fire a warning when a promise is created but not awaited on or returned. This ensures that all async operations are accounted for.

In the case where we don't care to wait for the promise to return (fire-and-forget scenario), using the `void` operator (which does nothing) is used to silence the warning by explicitly communicating that we don't care about when the promise resolves or what its result is.